extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -927,6 +943,29 @@ export class AmpStoryPage extends AMP.BaseElement {
       toggle(dev().assertElement(this.playMessageEl_), true));
   }
 
+  /**
+   * Renders the open attachment UI affordance.
+   * @private
+   */
+  renderAttachmentUI_() {
```

Comment:
I feel like we should call this `attachAttachmentUI`, since it doesn't actually render the UI, it just attaches it to the DOM, and listens to when it needs to be opened. Although my suggested name is a bit weird. WDYT?

Reply:
Maybe it's confusing because it may sound like we're rendering the attachment, when we're actually rendering that small message/button to open it?
`buildOpenAttachmentElement()` actually renders this button.
I renamed the method to `renderOpenAttachmentUI_()`, please lmk if you think this is more helpful